_posts/release/2022-12-01-accumulo-2.1.1.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Apache Accumulo 2.1.1
+sortableversion: '02.01.01'
+LTM: true
+draft: true
+---
+
+** DRAFT RELEASE NOTES **
+
+## About
+
+Apache Accumulo 2.1.1 is a patch release of the 2.1.0 LTM line. 
+
+### Major Improvements
+
+### Minor Improvements
+
+* {% ghi 3077 %} Call FSDataOutputStream.setDropBehind when reading and writing WAL files
+* {% ghi 3079 %} Call FSDataOutputStream.setDropBehind on major compaction input files
+* {% ghi 3023 %} Reduced the amount of logging when an ExternalCompaction is cancelled for cause
+
+### Behavior Changes
+
+* {% ghi 3082 %} ScanServers can now scan offline tables. The AccumuloClient will no longer
+  throw a TableOfflineException when creating a Scanner or BatchScanner against an offline table.
+  Instead, an error will be thrown on the call to iterator().
+
+### Bug Fixes
+
+* {% ghi 3059 %} Remove stale compactions from CompactionCoordinator internal structure
```

Comment:
The description here could clarify what bug was fixed that the user might have encountered (user impact narrative), rather than simply describing the patch we applied. I'm not super familiar with the user impact to know what to suggest.